docs/content/providers/marathon.md

```diff
@@ -490,7 +495,7 @@ providers:
 
 #### `tls.insecureSkipVerify`
 
-TODO add description.
+If `insecureSkipVerify` is `true`, TLS for the connection to Marathon accepts any certificate presented by the server and any host name in that certificate.
```

Comment:
```suggestion
Policy for certificate verification for the secured connection to Marathon.

If `insecureSkipVerify` is `true`, Traefik accepts any certificate presented by Marathon and any host name in that certificate.
```

Reply:
It's not related to Traefik, but to Marathon

Reply:
yes, this is the intent of my suggestion? Sounds like it doesn' t work: let's keep the original sentence then.